ipysankeywidget/sankey_widget.py

```diff
@@ -65,3 +74,20 @@ def save_png(self, filename):
     def auto_save_png(self, filename):
         self._auto_png_filename = filename
         return self
+
+    @observe("svg")
+    def _on_svg_data(self, change):
+        if change['type'] != 'change':
+            return
+        if self._auto_svg_filename:
+            self.save_svg(self._auto_png_filename)
+            self._auto_svg_filename = None
+
+    def save_svg(self, filename):
+        data = base64.decodebytes(bytes(self.png, 'ascii'))
```

Comment:
yep, didn't need to do anything, really, to the data, so just passing it directly